### PR TITLE
Feat/dropdown menu

### PR DESCRIPTION
**Pull request by guilhermecardoso-gympass**
[JIRA Issue](https://gympass.atlassian.net/browse/)

## Description 📄

Create a component Menu with Dropdown function!

## Platforms 📲

- [x] Web
- [ ] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [x] Snapshot Test

## Checklist: 🔍

- [x] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [x] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## Screenshots 📸

Screen Shot 2022-05-17 at 14 40 16

### CODE REVIEW

File: packages/yoga/src/Menu/web/MenuList.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  }) => {
+    return `
+    margin: ${menu.margin.default}px 0;
+    padding: 0;
+    min-width: ${menu.width.min}px;
+    max-width: ${menu.width.max}px;
+    border-radius: ${menu.border.radius}px;
+
+    background-color: ${menu.backgroundColor.white};
+    box-shadow: 0px 2px 6px rgba(152, 152, 166, 0.25);
```
Comment (caioalexandrebr) [inline packages/yoga/src/Menu/web/MenuList.jsx:27]: Great contribution @guilhermecardoso-gympass ! 🚀 

here should we use the [elevation token](https://gympass.github.io/yoga/guidelines/tokens/elevations)?

Comment (guilhermecardoso-gympass) [reply packages/yoga/src/Menu/web/MenuList.jsx:27]: this shadow not have in elevation token! at least I didn't find

Comment (georgia-gympass) [reply packages/yoga/src/Menu/web/MenuList.jsx:27]: caio is right about that gui, we should use the small elevation

Comment (guilhermecardoso-gympass) [reply packages/yoga/src/Menu/web/MenuList.jsx:27]: Fixed with a small elevation

**Review by ricardobylife — APPROVED** [review]

File: packages/yoga/src/hooks/useKeyPress.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    function upHandler({ key }) {
+      if (key === targetKey) {
+        setKeyPressed(false);
+      }
+    }
+    window.addEventListener('keydown', downHandler);
+    window.addEventListener('keyup', upHandler);
+
+    return () => {
+      window.removeEventListener('keydown');
```
Comment (davimdantas) [inline packages/yoga/src/hooks/useKeyPress.jsx:21]: Nice contribution, @guilhermecardoso-gympass 

removeEventListener should receive two params:
```suggestion
       window.removeEventListener('keydown', downHandler);
      window.removeEventListener('keyup', upHandler);
```

Where is this hook being used?

Comment (guilhermecardoso-gympass) [reply packages/yoga/src/hooks/useKeyPress.jsx:21]: Done!

**Pull request merged**